src/Mail/Helper/TextMessage.php

```diff
@@ -88,6 +88,7 @@ private function processPart($part): void
         $filename = $hasDisposition ? $part->getHeaderField('Content-Disposition', 'filename') : null;
         $is_attachment = $disposition === 'attachment' || $filename;
         $charset = $hasContentType ? $part->getHeaderField('Content-Type', 'charset') : null;
+        $content = (new DecodePart($part))->decode();
```

Comment:
is this safe to do? i.e decode unconditionally without checking type? I'm afraid the `(new DecodePart($part))->decode()` will either throw exception or produce garbage, for example for application/octet-steream of image/jpeg...

Reply:
`decode()` will return the original content unless there is a `Content-Transfer-Encoding` header set to either `quoted-printable` or `base64`. That should work independent of the part type. No reason why an image cannot be base64-encoded.

Currently `decode()` will throw an exception for unsupported encodings. That might be a problem, but not a new one.